src/main/java/com/faforever/client/update/ClientUpdateServiceImpl.java

```diff
@@ -94,8 +94,13 @@ public ComparableVersion getCurrentVersion() {
     return currentVersion;
   }
 
-  private void install(Path binaryPath) {
-    // TODO probably need to make this executable on unix See #1026
+  @VisibleForTesting
+  void install(Path binaryPath) {
```

Comment:
Please tell me why would we need to set the Executable bit again while installing if we already did so while downloading.
Or other question do we need to set it while downloading if we do during install...

Reply:
Even tho i am not sure what we are installing here? 🤔

Reply:
It is no the faf exe right?

Reply:
oh it is the client... XD ups... Well updating the client on linux does not work in the first place becasue it is a tar.gz and we treat it as a exe just so you know. Something that needs to be fixed as well ;) But yeah you can set the executable bit there i guess even tho I would assume a tar.gz does not need one :D

Reply:
@tatsujb as a install script maybe one day we can somehow incorporate that